### DISCUSSION TITLE
#help |  Custom element doesn't exist: bubble-pop-up

### DISCUSSION THREAD

**Discussion by labaland** [Clooos/Bubble-Card#489] [Questions about config, custom styles and templates] [answered]
Hi! Just out of nowhere this started to show up on all devices. I tried to reinstall in hacs, any1 knows why?
Only problem with bubble! :( 

thanks! :) 

![image](https://github.com/Clooos/Bubble-Card/assets/20958435/02044dd1-c337-4df5-be41-ddc9e82b68a3)
![image](https://github.com/Clooos/Bubble-Card/assets/20958435/1cdf7981-641b-453e-ac60-e2fa5d1e8b4b)

**Comment by Clooos** [comment]
Hi, can you take a look at your HA logs to see if there is any Bubble Card related error?

**Reply by labaland** [reply]
Hi! There is no errors there for me. And this started 1 day after i updated and tried the beta for the first time. But now im back on latest and not beta. if i try to re install again, is there anything more i can do more than just press uninstall/remove in hacs?

**Reply by labaland** [reply]
Got it to work now after i added the 2 /local.... lines to resources, i only got the first row by hacs.
Only the first row should be enough if it was working as it should?
sry bad english!

thx!
![image](https://github.com/Clooos/Bubble-Card/assets/20958435/0163852f-046a-4d41-b42b-8825b4b2fd24)

**Reply by Clooos** [reply, marked as answer]
I've not read your issue correctly, this is because you are still using the previous bubble-pop-up.js, it was removed a few version back. 

You can now change your `custom:bubble-pop-up` to `custom:bubble-card` then add `card_type: pop-up` to all of them. Then you can remove it.

More details here:
https://github.com/Clooos/Bubble-Card/?tab=readme-ov-file#pop-up-optimization

**Reply by labaland** [reply]
Thank you!
One more fast question, i have this Swipe card on my dashboard ( https://github.com/bramkragten/swipe-card?tab=readme-ov-file )
but when i use a popup by your card that card is still visible above/with the popup, anything that is possible to fix? :P 

Popup closed:
![image](https://github.com/Clooos/Bubble-Card/assets/20958435/6775bed9-6ded-4677-85c5-c798c5977e2e)

Popup opened:
![image](https://github.com/Clooos/Bubble-Card/assets/20958435/30948546-880d-462d-be66-f9eda51eb8ce)

**Reply by Clooos** [reply]
Can you try the new Bubble Card beta? You can get it by going on "Redownload" on the Bubble Card HACS page, this issue should be fixed on that version.

**Reply by labaland** [reply]
Thanks again! That fixed it! 👍 :D
